### PR TITLE
Refactor cli - depend on aleph-sdk-python and remove code duplication

### PR DESCRIPTION
**Pull request by aliel**

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -30,13 +30,14 @@ package_dir =
 # DON'T CHANGE THE FOLLOWING LINE! IT WILL BE UPDATED BY PYSCAFFOLD!
 setup_requires = pyscaffold>=3.2a0,<3.3a0
 # Add here dependencies of your project (semicolon/line-separated), e.g.
-install_requires = 
+install_requires =
+    aleph-sdk-python@git+https://github.com/aleph-im/aleph-sdk-python.git@main
```
Comment (MHHukiewitz) [inline setup.cfg:34]: shouldn't it be the current version on PyPI instead?

**Review by MHHukiewitz — COMMENTED** [review]
This currently doesn't work, if you install the latest version of `aleph-sdk-python` you still access all the modules through `aleph_client`. Is there some other PR on aleph-sdk-python that I'm not aware of?

File: setup.cfg (hunk 1 above)
Comment (hoh) [reply setup.cfg:34]: The version on PyPI is outdated, use the `master` from git for now.

I will upgrade it this week.

**Comment by aliel** [comment]
>  you still access all the modules through `aleph_client`

no
> Is there some other PR on aleph-sdk-python that I'm not aware of?

no

it should work on branch "aliel-refactor-sdk-146"

**Comment by MHHukiewitz** [comment]
Right, after the latest update of aleph-sdk-python, it works.

**Comment by hoh** [comment]
The refactoring is incomplete and should not have been merged, see #150 to follow up on this.

**Pull request merged**